tests/unit/utils/Utils.h

```diff
@@ -0,0 +1,15 @@
+#ifndef __UTILS_H__
+#define __UTILS_H__
+
+/*
```

Comment:
These files need BSD headers (unless they are GPL, in which case we cannot use them)